### PR TITLE
Update netatmo.markdown

### PR DESCRIPTION
**Pull request by cgtobi**
**Description:**
Remove documentation about (temporarily) removed features and configuration options.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#29851

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by fabaff — APPROVED** [review]

File: source/_integrations/netatmo.markdown [hunk 1]
```diff
@@ -30,45 +30,27 @@ To enable the Netatmo component, add the following lines to your `configuration.
...
-api_key:
+client_id:
   description: The `client id` from your Netatmo app.
   required: true
   type: string
-secret_key:
+client_secret:
   description: The `client secret` from your Netatmo app.
   required: true
   type: integer
```
Comment (rugaard) [inline source/_integrations/netatmo.markdown:45]: I know I'm late to the party, but I'm pretty confused from the docs.

How can this ever be an integer? It's always be a `string`. It's even a string in the provided screenshot?

Comment (cgtobi) [reply source/_integrations/netatmo.markdown:45]: @rugaard well spotted. Thanks. Do you mind fixing it and opening a PR?

Comment (rugaard) [reply source/_integrations/netatmo.markdown:45]: Done  https://github.com/home-assistant/home-assistant.io/pull/12009  👍

Comment (cgtobi) [reply source/_integrations/netatmo.markdown:45]: Great. Thanks!

**Pull request merged**